bindings/pyroot/JupyROOT/helpers/utils.py

```diff
@@ -375,8 +374,7 @@ def _getUID(self):
         Every DIV containing a JavaScript snippet must be unique in the
         notebook. This methods provides a unique identifier.
```

Comment:
Can you extend the documentation of `_getUID` to explain why we need to create the identifier like this (i.e. explain JupyterLab case)?